kits/rosie/rosie_demo_utils.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+// HEBI C++ API files:
+#include "arm/arm.hpp"
+#include "group_command.hpp"
+#include "group_feedback.hpp"
+#include "lookup.hpp"
+#include "util/mobile_io.hpp"
+#include "group.hpp"
+#include "color.hpp"
+#include <thread>
+#include <chrono>
+
+// Common includes
+#include <iostream>
```

Comment:
Unless the `#include` files are used in the header, it is best to just include them in the `.cpp` file instead.  This is because having these in the header means every .cpp file that includes this one _also_ includes the full contents of these `include`s, leading to potentially a lot more lines to compile.

On a project of this size, this hardly matters, but I've seen larger projects where carefully pruning the `#include` to only necessary ones can save 10-20 minutes on a 40 minute build!

(This can be further optimized by forward declaring classes that are only used by reference in here, but I dont think that's necessary)